tools/tpm2_import.c

```diff
@@ -312,18 +317,30 @@ static bool on_option(char key, char *value) {
     case 'K':
         ctx.parent_key_public_file = value;
         break;
+    case 'k':
+        ctx.input_enc_key_file = value;
+        break;
     case 'u':
-        ctx.import_key_public_file = value;
+        ctx.public_key_file = value;
         break;
     case 'r':
-        ctx.import_key_private_file = value;
+        ctx.private_key_file = value;
         break;
     case 'b':
         ctx.attrs = value;
         break;
     case 'g':
         ctx.name_alg = value;
         break;
+    case 's':
+        ctx.input_seed_file = value;
+        break;
+    case 'd':
+        ctx.input_duplicate_file = value;
```

Comment:
can you re-use -i for this and auto-detect the format?

Reply:
will do, I have been trying to keep my fingers off the original code as much as possible.